dataProviders/WorldDungeonEntranceDataProvider.lua

```diff
@@ -90,6 +91,26 @@ function WorldDungeonEntrancePinMixin:UpdateMousePropagation() end
 function WorldDungeonEntrancePinMixin:DoesMapTypeAllowSuperTrack() return true; end
 
 local teleportInstructionText = '<' .. StripHyperlinks(WARDROBE_SHORTCUTS_TUTORIAL_2):gsub('[[%]]', '') .. ': ' .. TELEPORT_TO_DUNGEON .. '>';
+local tomTomInstructionText = '<Alt Click to set TomTom waypoint>'
+
+function WorldDungeonEntrancePinMixin:OnMouseClickAction(button)
+    if button == "LeftButton" and TomTom and IsAltKeyDown() then
+        TomTom:AddWaypoint(self:GetMap():GetMapID(), self.poiInfo.position.x, self.poiInfo.position.y, {
+            title = self.name,
+            from = AddOnFolderName,
+            persistent = nil,
+            minimap = true,
+            world = true
+        })
+    else
```

Comment:
might wanna `elseif button == "LeftButton"` this 🙂

Reply:
The reason why I didn't is because the base code is just 
```lua
function DungeonEntrancePinMixin:OnMouseClickAction(button)
    SuperTrackablePinMixin.OnMouseClickAction(self, button);

	if button == "RightButton" then
		EncounterJournal_LoadUI();
		EncounterJournal_OpenJournal(nil, self.journalInstanceID);
	end
end
```
So I figured that I'd let it bubble up to `SuperTrackablePinMixin` if it doesn't fall into the TomTom code regardless of what the _reason_ for the fail was 🤔

Reply:
fair enough ^^